pkg/apis/autoscaling/v1alpha1/metric_types.go

```diff
@@ -65,6 +65,8 @@ type MetricSpec struct {
 	PanicWindow time.Duration `json:"panicWindow"`
 	// ScrapeTarget is the K8s service that publishes the metric endpoint.
 	ScrapeTarget string `json:"scrapeTarget"`
+	// Algorithm is the metric averaging algorithm.
+	Algorithm string `json:"algorithm"`
```

Comment:
Should probably be at least AggregationAlgorithm

Reply:
Done.